src/Components/AMPM.js

```diff
@@ -0,0 +1,17 @@
+function AMPM(hours) {
```

Comment:
does JavaScript pass by reference or value?  If it is only passed by value Im not sure this will work.  That will also explain why SonarCloud is giving you code smells

Reply:
I saw the smells. i will clean them up. it must pass by value. i assumed it was more like Python but i guess it is more like C.

Reply:
great change.  thanks for fixing the thing i introduced haha